### PR TITLE
Component to search organisation

### LINKED ISSUE #33: Component to search organisation UI/UX
**Is your feature request related to a problem? Please describe.**
A component for the user to search for organisations.

**Describe the solution you'd like**
`components/organisation/searchOrg.js`
A class-based component with a search box which takes in an array of objects (organisations and their ids).

It has three states.
`components/organisation/searchOrg.stories.js`
1) Initial state
2) loading state
3) Search State
4) Error state( In case the length of the array is 0 it shows "No Organisation found".)

**Additional context**
Screenshot from 2020-06-11 08-23-19

### PR DESCRIPTION
**Pull request by Nikhil-dev13**
### Description
Created a class based component to list all organisations and search organisations with four states
1)Initial State
2)Loading State
3)Error State
4)Search State
5)No Search Results Returned
**Type of change** 

- [x] New feature (non-breaking change which adds functionality pre-approved by mentors)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor
- [ ] Addition of testcases
- [ ] This change requires a documentation update (software upgrade on readme file)

### Screenshots of change:
1)Initial State
image

2)Loading State
image

3)Error State
image

4)Search State
image

5)When no Search Results are returned
image

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] I have updated storybook
- [ ] I have updated README.md
- [ ] Added new npm packages

**Code/Quality Assurance**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] New and existing tests pass locally with my changes

### CODE REVIEW

**Comment by pratik0204** [comment]
Don't hide the search bar if no organisation is found @Nikhil-dev13

**Comment by pratik0204** [comment]
Also update the ss.

File: src/components/organisation/searchOrg.stories.js [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+		},
+	],
+	loading: false,
+	search: [],
+	search_value: '',
+	search_flag: false,
+};
+
+export const loadingStateData = {
+	...initialStateData,
```
Comment (pratik0204) [inline src/components/organisation/searchOrg.stories.js:59]: there won't be any org in array if loading is true, when loading is done, then state will be filled wilth data, so no need of this line.

Comment (Nikhil-dev13) [reply src/components/organisation/searchOrg.stories.js:59]: ok

File: src/components/organisation/searchOrg.js [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+								name='search_value'
+								onChange={this.inputChangeHandler}
+								value={search_value}
+								style={classes.search_bar}
+								variant='outlined'
+								/*Search Icon*/
+								InputProps={{
+									startAdornment: (
+										<InputAdornment>
+											<IconButton>
```
Comment (kaushik27mishra) [inline src/components/organisation/searchOrg.js:156]: No need of Icon button here.

File: src/components/organisation/searchOrg.js [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+				</Grid>
+				{/*Search Bar*/}
+				<Grid item style={classes.root}>
+					<Grid
+						container
+						direction='row'
+						alignItems='center'
+						spacing={2}>
+						<Grid item>
+							<TextField
```
Comment (kaushik27mishra) [inline src/components/organisation/searchOrg.js:143]: In the loading state, the text filed should be disabled with a value in it.

File: src/components/organisation/searchOrg.js (hunk 2 above)
Comment (Nikhil-dev13) [reply src/components/organisation/searchOrg.js:156]: Ok Sir

File: src/components/organisation/searchOrg.js (hunk 3 above)
Comment (Nikhil-dev13) [reply src/components/organisation/searchOrg.js:143]: image

Something like this ....?

File: src/components/organisation/searchOrg.js (hunk 2 above)
Comment (Nikhil-dev13) [reply src/components/organisation/searchOrg.js:156]: done

File: src/components/organisation/searchOrg.js (hunk 3 above)
Comment (kaushik27mishra) [reply src/components/organisation/searchOrg.js:143]: Yes!

**Comment by Nikhil-dev13** [comment]
@kaushik27mishra done sir

**Comment by Nikhil-dev13** [comment]
all the requested changes are done @kaushik27mishra

**Pull request merged**